Assignment6/CParkingLot.cpp

```diff
@@ -0,0 +1,95 @@
+#include "CParkingLot.h"
+
+
+CParkingLot::CParkingLot() {
+    std::cout << "How many places are in the parking lot?" << std::endl;
+    std::cin >> NumberOfPlaces_;
+
+    if (NumberOfPlaces_ > 0) {
+        std::cout << "Describe the parking lot" << std::endl;
+    }
+    get_parking_information(PlacesToCars_, CarsToPlaces_);
```

Comment:
No:)
There'll be a mistake if I put it in the if-block, because in this case, if NumberOfPlaces_=0, PlacesToCars_ and CarsToPlaces_ won't be declared.
So I need to execute this function anyway